test/unittests/Microsoft.ServiceFabric.Actors.Tests/AsemlbyAttributes.cs

```diff
@@ -0,0 +1,5 @@
+using Xunit;
+
+// This attribute is used to run the tests in the same assembly in sequence.
```

Comment:
It would be great to explain WHY the ActorRemotingProviderAttributeTest methods need to run sequentially. I.e. something like `// Run tests sequentially because ActorRemotingProviderAttribute has static state and cannot be tested in parallel`